### ISSUE TITLE
FEATURE REQUEST: signalP6.0 support

### ISSUE THREAD

**Issue by xizhesun** [ccdmb/predector#43]
Dear Darcy,

SignalP was updated to version 6.0 and should be better than the singnalP5.0. I think we could support it and might get a better result.

Thanks,
Xizhe

**Comment by xizhesun** [comment]
And also deepTMHMM support. Thanks!

**Comment by xizhesun** [comment]
Sorry, I have found the guide for installation. So I change the issue title.

**Comment by darcyabjones** [comment]
Thanks for suggesting these.
They hadn't come to my attention yet (i'm doing slightly un-related things now).

Working on getting a SignalP 6 installer at the moment.

I think we might need to wait and see what happens with DeepTMHMM though.
Currently the only access to it is via their biolib thing and there isn't a paper or any other description of what it does.

I'm not very familiar with biolib, but from what i've just run it seems to download the program and run it via docker, which would be a problem for us.
Basically anyone using a supercomputer wouldn't be able to use it, and we'd have to do some pretty nasty workarounds if we still want the pipeline to support docker containers (because we shouldn't spawn a container within a container).

Anyway.
SignalP6 is on it's way.
I'll keep an eye on DeepTMHMM to see what happens, but maybe not yet.

**Comment by xizhesun** [comment]
Thanks Darcy! Actually, the current version is enough for me. But I still want Predector to be better.

**Comment by darcyabjones** [comment]
Hey Xizhe,,

Sorry for the delay.
The latest version should now run SignalP 6 and EffectorP 3 (1.1.0-alpha).

I've done some testing on the update and it seems to be ok, but I need to try a few other operating systems before I call it stable.
Documentation on the main README is up to date but the wiki needs an update too.

Just make sure to run `nextflow drop ccdmb/predector` if you're running it on a computer that has run the pipeline before.
This just makes sure that nextflow pulls in the latest changes (i've had some issues with nextflow update).
Will add some more docs on this soon.

I'll close this and the EffectorP issues, but please do re-raise it or start a new issue if you have problems.

Cheers, Darcy
